core/drivers/crypto/crypto_api/acipher/ecc.c

```diff
@@ -98,19 +98,22 @@ static TEE_Result ecc_generate_keypair(struct ecc_keypair *key,
 {
 	TEE_Result ret = TEE_ERROR_NOT_IMPLEMENTED;
 	struct drvcrypt_ecc *ecc = NULL;
+	size_t key_size = 0;
```

Comment:
for consistency, prefer rename to `key_size_bits`.

Reply:
I agree